tensorboard/components/tf_paginated_view/tf-category-paginated-view.html

```diff
@@ -176,13 +193,17 @@
 
       .category-name {
         white-space: pre;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        padding: 2px 0;
       }
 
       .count {
-        float: right;
-        margin-right: 5px;
+        margin: 0 5px;
         font-size: 12px;
         color: var(--paper-grey-500);
+        display: flex;
+        align-items: center;
```

Comment:
you may want to add flex: 0; (also consider using a-z :) )

Reply:
Done as flex: none.

Deferring a-z to another day.